OpenKeychain/src/main/res/xml/keyserver_sync_adapter_desc.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<sync-adapter xmlns:android="http://schemas.android.com/apk/res/android"
+    android:contentAuthority="org.sufficientlysecure.keychain.debug.provider"
```

Comment:
Done in https://github.com/open-keychain/open-keychain/commit/d461b424d2ddb7b71a73568fcf006c2a09829768